web-registry/src/lib/metadata-graph.ts

```diff
@@ -0,0 +1,11 @@
+import axios, { AxiosResponse } from 'axios';
+
+export const getMetadata = async (hash: Uint8Array): Promise<AxiosResponse> => {
+  if (hash) {
+    // TODO: this is a POC for getting bytes from the ledger and decoding to fetch from DB.
+    // In the future, we will use our custom iri-gen script to manage the hashes, not atob().
```

Comment:
I don't think this comment is relevant.
The iri-gen script doesn't change anything to that, it just gives you the hashed iri for some JSON-LD data, that you can then use in the on chain `metadata` field
But since this metadata field is still bytes, the string will need to be base64 encoded in order to store it on-chain, that's it.

Maybe we should just replace it with the following:
```suggestion
The on-chain metadata fields being stored as bytes for now, we need to decode it using base64 to get the corresponding IRI value as a string.
This won't be needed once the on-chain metadata fields are converted to string: https://github.com/regen-network/regen-ledger/issues/829
```